nextflow.config

```diff
@@ -1,85 +1,145 @@
-// =================================================================
-// nextflow.config is the main configuration script for a nextflow pipeline
-// Should contain the following sections:
-	// A manifest for defining workflow metadata
-	// Mandated minimal version of Nextflow required to run pipeline
-	// Default workflow parameter definitions
-	// Shell behaviour settings for the workflow
-	// Execution reports
-	// Configuration profiles
-	// Default resource definitions for processes
-
-// Examples are included for each section. Remove them and replace
-// with project-specific code. For more information see:
-// https://www.nextflow.io/docs/latest/index.html.
-//
-// ===================================================================
-
-// Define some metadata for publishing pipeline on GitHub
-// See https://www.nextflow.io/docs/latest/config.html#scope-manifest
+// Workflow details
 manifest {
-	author = 'Georgie Samaha'
-	name = 'Nextflow_DSL2_template-nf'
-	description = 'Template for creating Nextflow workflows'
-	homePage = 'https://github.com/Sydney-Informatics-Hub/Nextflow_DSL2_template'
-}
+    
+  author = ' Tracy Chew, Cali Willet, Nandan Deshpande, Sydney Informatics Hub, University of Sydney'
+  name = 'SomaticShortV-nf'
+  description = 'GATK somatic short variant calling pipeline'
+  homePage = 'https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf'
+    }
 
 // Mandate a minimum version of nextflow required to run the pipeline
-// See https://www.nextflow.io/docs/latest/config.html#nextflow-version
-nextflowVersion = '!>=20.07.1'
+nextflowVersion     = '!>=20.07.1'
 
-// Set default workflow parameters
-// See https://www.nextflow.io/docs/latest/config.html#scope-params 
+// Set default parameters
 params {
-	help	= false
-	outDir	= 'results'
-	input	= false
+  version                       = '1.0'
+  help                          = false
+  input                         = null
+  ref                           = false
+  intervals                     = false
+  outDir		        = './results'
+
+
+  // Intervals for 'Scatter-Gather'
+  intervalList          = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n']
+  intervalList_path = false
+
+  // PATH to PoN (created previously)
+  //params.path_to_PON          = 'PoN_folder'
+  //params.ponvcf               = params.path_to_PON+'/'+'pon.vcf.gz'
+
+  
+  gadi_account       	= false
+  setonix_account     	= false
+  whoami            	= false
+}
+
+params.common_biallelic_path = params.ref+'/gatk-best-practices/somatic-hg38/'+'small_exac_common_3.hg38.vcf.gz'
```

Comment:
why is this params sitting outside the params scope?

Reply:
This param is not required to be set here. Removed now.